releases/release-6.6.0.md

```diff
@@ -0,0 +1,440 @@
+---
+title: TiDB 6.6.0 Release Notes
+---
+
+# TiDB 6.6.0 Release Notes
+
+Release date: xx, 2023
+
+TiDB version: 6.6.0-DMR
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v6.6/quick-start-with-tidb) | [Installation package](https://cn.pingcap.com/product-community/)
+
+In v6.6.0-DMR, the key new features and improvements are as follows:
+
+- MySQL 8.0 兼容的多值索引 (Multi-Valued Index) (实验特性)
+- 基于资源组的资源管控 (实验特性)
+- 悲观锁队列的稳定唤醒模型
+- 数据请求的批量聚合
+
+## New features
+
+### SQL
+
+* Support dynamically managing the resource usage of DDL operations (experimental) [#38025](https://github.com/pingcap/tidb/issues/38025)  @[hawkingrei](https://github.com/hawkingrei) **tw@ran-huang**
+
+    TiDB v6.6.0 introduces resource management for DDL operations to reduce the impact of DDL changes on online applications by automatically controlling the CPU usage of these operations. This feature is effective only after the [DDL distributed parallel execution framework](/system-variables.md#tidb_ddl_distribute_reorg-new-in-v660) is enabled.
+
+* Support the MySQL-compatible foreign key constraints [#18209](https://github.com/pingcap/tidb/issues/18209) @[crazycs520](https://github.com/crazycs520) **tw@Oreoxmt**
+
+    TiDB v6.6.0 introduces the foreign key constraints feature, which is compatible with MySQL. This feature supports referencing within a table or between tables, constraints validation, and cascade operations. This feature helps to maintain data consistency, improve data quality, and facilitate data modeling.
```

Comment:
```suggestion
    TiDB v6.6.0 introduces the foreign key constraints feature, which is compatible with MySQL. This feature supports referencing within a table or between tables, constraints validation, and cascade operations. This feature helps to migrate applications to TiDB, maintain data consistency, improve data quality, and facilitate data modeling.
```

1. I think application migration and correct ORM support are important to name here.
2. I think foreign keys is the most important feature of this release. Maybe list it as the first item or so?

Reply:
> I think foreign keys is the most important feature of this release. Maybe list it as the first item or so?

@dveeden After confirming with PM @easonn7, the **resource management** feature is also important. I add foreign keys as a **key new feature** [here](https://github.com/pingcap/docs/pull/12321/files#r1100946919)